src/common/components/nav/nav.component.js

```diff
@@ -174,20 +174,11 @@ class NavController{
     }))
       .map((response) => {
         let jsonStructure = response.data;
-        let menuStructure = {
-          main: jsonStructure['/content/cru/us/en'],
+        return {
+          main: jsonStructure['/content/cru/us/en'] || jsonStructure['main'],
           global: jsonStructure['/content/cru/us/en/global'],
           give: jsonStructure['/content/give/us/en']
         };
-
-        //add give to main nav
-        menuStructure.main.push({
-          title: 'Give',
-          path: '/give',
-          children: jsonStructure['/content/give/us/en']
-        });
```

Comment:
I think this code section is still required if the jsonStructure var doesn't contains a "main" component. what do you think?